.github/workflows/lint_code.yml

```
@@ -46,4 +46,6 @@ jobs:
         id: checkov
         uses: bridgecrewio/checkov-action@master
         with:
-          args: --quiet --framework terraform
+          quite: true
```

typo
```suggestion
          quiet: true
```

---

let's also add the option `evaluate-variables: true` to ensure the default values of the variables are taken into account when anylyzing
https://www.checkov.io/2.Basics/Handling%20Variables.html

---

I think it's better to use a configuration file instead of passing all options to this Github Action 
https://github.com/bridgecrewio/checkov#configuration-using-a-config-file

```suggestion
       with:
          config_file: ".checkov.yaml"
```

---

The typo was fixed in 8017c42, and also separate configurations into file in 89d4708 ✅ ✨